README.md

```diff
@@ -2,14 +2,14 @@
 
 Barcelona on Rails is dedicated to providing a harassment-free user group experience for everyone, regardless of sex, sexual orientation, gender identity, disability, physical appearance, body size, race, religion or nationality. We do not tolerate harassment of participants in any form.
 
-Harassment includes offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention.
+Harassment includes offensive verbal comments related to sex, sexual orientation, gender identity, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention.
```

Comment:
That makes it consistent again :+1: 

Oh, I forgot to mention that I changed  `... race, or religion.` to `... race, religion or nationality.`.
So we should include `nationality` here as well, between `religion, sexual images`.

And I wonder if we need `in public spaces`, or if we should remove it.

Reply:
Are you talking about "sexual images in public spaces"?

Pros of removing it:

1. We don't have to debate whether Xing's lounge is a public space or not.

Cons:

1. Since someone wrote that, there's a chance that they thought of a case where sexual images are OK, and that's why they didn't want a blanket ban.  Sure, we are not trying to stop people from having nude pics on their phones.  We just don't want them to share them with others at our meetings.

So I think we should remove it but I fear I'm missing something that the original author thought of.

Reply:
To me `in public spaces` leads to questions, not to clarifications. That's why I think it is unnecessary to harmful.